### ISSUE TITLE
scrape() should take object of options, not ordered parameters

### ISSUE THREAD

**Issue by tobiasmuehl** [sudheer-ranga/aliexpress-product-scraper#5]
Seeing that there are more features added that need input parameters, I think we should refactor the scrape function as such:

```typescript
type AliExpressScrapeOptions = {
  puppeteerOptions?: LaunchOptions,
  feedbackLimit?: Number,
  shippingCountryNames?: String[] // should be typed with the list of country codes so it's clear which standard is being used
}

function scrape(productId: number, options: AliExpressScrapeOptions = {})

```

**Comment by sudheer-ranga** [comment]
Yes @tobiasmuehl. This will be taken care of in the next release.

**Comment by MrBaz** [comment]
Hi, thanks for the great efforts. 
I have a question please, can you explain what is _await page.evaluate(() => runParams);_ do and what is the runParams value ?

**Comment by sudheer-ranga** [comment]
> Hi, thanks for the great efforts.
Thank you for your kind words.

> I have a question please, can you explain what is _await page.evaluate(() => runParams);_ do and what is the runParams value ?

```await page.goto(`https://www.aliexpress.com/item/${productId}.html`);
const aliExpressData = await page.evaluate(() => runParams);```

1. Visits the given URL and wait till the page is loaded.
2. Once the page is loaded, we will search for the variable `runParams` and return its value which is an object. We are using this object info to build JSON.

**Comment by dilazure** [comment]
Can you explain how it use?

**Comment by sudheer-ranga** [comment]
This is fixed in the new release v2.0.2. Closing the issue.
